Date/TimeZone.php

```diff
@@ -311,11 +311,10 @@ class Date_TimeZone
      *
      * @param string $ps_id the time zone ID
      *
-     * @return   void
      * @access   public
      * @see      Date::setTZ(), Date::setTZByID(), Date_TimeZone::isValidID()
      */
-    function Date_TimeZone($ps_id)
+    function __construct($ps_id)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.